File: CS/WebSolution.Module.Web/CustomizeASPxTabControlViewController.cs
using System;
using DevExpress.ExpressApp;
using System.Web.UI;
using DevExpress.ExpressApp.Editors;
using DevExpress.Web.ASPxTabControl;
using System.Drawing;
using DevExpress.ExpressApp.Layout;
using DevExpress.ExpressApp.Web.Layout;
using DevExpress.ExpressApp.Model;

namespace MainDemo.Module.Web {
    public partial class CustomizeASPxTabControlViewController : ViewController {
        public CustomizeASPxTabControlViewController() {
            InitializeComponent();
            RegisterActions(components);
            TargetViewType = ViewType.DetailView;
        }
        protected override void OnActivated() {
            base.OnActivated();
            ((WebLayoutManager)((DetailView)View).LayoutManager).ItemCreated += new EventHandler<ItemCreatedEventArgs>(CustomizeASPxTabControlViewController_ItemCreated); 
        }
        void CustomizeASPxTabControlViewController_ItemCreated(object sender, ItemCreatedEventArgs e) {
            if (e.ModelLayoutElement is IModelTabbedGroup) {
                CustomizeASPxTabControl(FindFirstControl<ASPxTabControlBase>(e.TemplateContainer));
            }
        }
        private T FindFirstControl<T>(Control parent) where T : Control {
            foreach (Control control in parent.Controls) {
                if (control is T) {
                    return (T)control;
                }
            }
            foreach (Control control in parent.Controls) {
                return FindFirstControl<T>(control);
            }
            return null;
        }
        protected override void OnDeactivated() {
            ((WebLayoutManager)((DetailView)View).LayoutManager).ItemCreated -= new EventHandler<ItemCreatedEventArgs>(CustomizeASPxTabControlViewController_ItemCreated); 
            base.OnDeactivated();
        }
        private void CustomizeASPxTabControl(ASPxTabControlBase tabControl) {
            tabControl.ActiveTabStyle.Border.BorderStyle = System.Web.UI.WebControls.BorderStyle.Solid;
            tabControl.ActiveTabStyle.Border.BorderWidth = 5;
            tabControl.ActiveTabStyle.Border.BorderColor = Color.Red;
        }
    }
}

File: VB/WebSolution.Module.Web/CustomizeASPxTabControlViewController.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.ExpressApp
Imports System.Web.UI
Imports DevExpress.ExpressApp.Editors
Imports DevExpress.Web.ASPxTabControl
Imports System.Drawing

Namespace MainDemo.Module.Web
	Partial Public Class CustomizeASPxTabControlViewController
		Inherits ViewController
		Public Sub New()
			InitializeComponent()
			RegisterActions(components)
			TargetViewType = ViewType.DetailView
		End Sub
		Protected Overrides Sub OnActivated()
			MyBase.OnActivated()
			AddHandler View.ControlsCreated, AddressOf View_ControlsCreated
		End Sub
		Protected Overrides Sub OnDeactivating()
			RemoveHandler View.ControlsCreated, AddressOf View_ControlsCreated
			MyBase.OnDeactivating()
		End Sub
		Private Sub View_ControlsCreated(ByVal sender As Object, ByVal e As EventArgs)
			AddHandler (CType(View.Control, Control)).PreRender, AddressOf CustomizeASPxTabControlViewController_PreRender
		End Sub
		Private Sub CustomizeASPxTabControlViewController_PreRender(ByVal sender As Object, ByVal e As EventArgs)
			RemoveHandler (CType(View.Control, Control)).PreRender, AddressOf CustomizeASPxTabControlViewController_PreRender
			Dim detailView As DetailView = CType(View, DetailView)
			For Each editor As ListPropertyEditor In detailView.GetItems(Of ListPropertyEditor)()
				Dim tabControl As ASPxTabControlBase = TryCast(FindASPxTabControl(TryCast(editor.Control, Control)), ASPxTabControlBase)
				If tabControl IsNot Nothing Then
					CustomizeASPxTabControl(tabControl)
				End If
			Next editor
		End Sub
		Private Function FindASPxTabControl(ByVal control As Control) As Control
			If control Is Nothing OrElse TypeOf control Is ASPxTabControlBase Then
				Return control
			Else
				Return FindASPxTabControl(control.Parent)
			End If
		End Function
		Private Sub CustomizeASPxTabControl(ByVal tabControl As ASPxTabControlBase)
			tabControl.ContentStyle.Border.BorderStyle = System.Web.UI.WebControls.BorderStyle.Solid
			tabControl.ContentStyle.Border.BorderWidth = 5
			tabControl.ContentStyle.Border.BorderColor = Color.Red
		End Sub
	End Class
End Namespace